addons/mail/static/tests/discuss/discuss_tests.js

```diff
@@ -61,6 +61,40 @@ QUnit.module("mail", (hooks) => {
         );
     });
 
+    QUnit.test("can change the thread name of #general", async (assert) => {
+        const server = new TestServer();
+        server.addChannel(1, "general", "General announcements...");
+        const env = makeTestEnv((route, params) => {
+            if (
+                route.startsWith("/mail") ||
+                ["/web/dataset/call_kw/mail.channel/channel_rename"].includes(route)
+            ) {
+                assert.step(route);
+            }
+            return server.rpc(route, params);
+        });
+        await mount(Discuss, target, { env });
+        env.services["mail.messaging"].setDiscussThread(1);
+
+        assert.containsOnce(target, ".o-mail-discuss-thread-name");
+        const threadNameElement = $(target).find(".o-mail-discuss-thread-name")[0];
+
+        await click(threadNameElement);
+        assert.strictEqual(threadNameElement.value, "general");
+        await editInput(target, ".o-mail-discuss-thread-name", "special");
+        await triggerEvent(target, ".o-mail-discuss-thread-name", "keydown", {
+            key: "Enter",
+        });
+        assert.strictEqual(threadNameElement.value, "special");
+
+        assert.verifySteps([
+            "/mail/init_messaging",
+            "/mail/inbox/messages",
```

Comment:
This test should not care for `mail/inbox/messages`.
It's best not having it in test, because if we change this, it's not great to adapt all tests that where asserting more than they should.